Documentation/upgrade.md

```diff
@@ -314,3 +314,12 @@ kubectl -n rook get pod -l app=rook-ceph-mds -o jsonpath='{range .items[*]}{.met
 
 ## Completion
 At this point, your Rook cluster should be fully upgraded to running version `rook/rook:v0.5.1` and the cluster should be healthy according to the steps in the [health verification section](#health-verification).
+
+## Upgrading Kubernetes
+:warning:  
```

Comment:
This warning logo appears on its own line. Perhaps we should add a sentence beside it. Can you add that this is only relevant if we were originally using K8s 1.6 or older? If the user is migrating from 1.7 to 1.8, this is not relevant. Perhaps a one-sentence title should make it more clear